js/script.js

```diff
@@ -1,290 +1,127 @@
-    return obj ? Object.keys(obj)
-      .map(function(key) {
-        var val = obj[key];
-        if (Array.isArray(val)) {
-          return val.map(function(val2) {
-            return encodeURIComponent(key) + '=' + encodeURIComponent(val2);
-          })
-            .join('&');
-        }
-        return encodeURIComponent(key) + '=' + encodeURIComponent(val);
-      })
-      .join('&') : '';
-  };
-  queryString.push = function(key, new_value) {
-    var params = queryString.parse(location.search);
-    if (new_value == null) {
-      delete params[key];
-    } else {
-      params[key] = new_value;
-    }
-    var new_params_string = queryString.stringify(params);
-    history.pushState({}, "", window.location.pathname + '?' + new_params_string);
-  };
-  if (typeof module !== 'undefined' && module.exports) {
-    module.exports = queryString;
-  } else {
-    window.queryString = queryString;
-  }
-})();
-$("select")
-  .change(function() {
-    console.log($(this)
-      .val());
-    var selectedState = $(this)
-      .val();
-    queryString.push('state', selectedState);
-
-//    window.location.reload();
-    window.location.replace(location.protocol + '//' + location.host + location.pathname +"?state="+selectedState);
-
-    //  populateDropDown(state);
-
-  });
-$("img")
-  .error(function() {
-    $(this)
-      .hide();
-    // or $(this).css({visibility:"hidden"});
-  });
-Handlebars.registerHelper('breaklines', function(text) {
-  text = Handlebars.Utils.escapeExpression(text);
-  //some legislators, such as VAL000157 have successive newlines in the offices.address fields
-  text = text.replace(/(\n ?\n)/gm, '\n');
-  //replace "\n" with "<br />"
-  text = text.replace(/(\r\n|\n|\r)/gm, '<br />');
-  return new Handlebars.SafeString(text);
 });
-var app = {};
 
-function init() {
-  var state1 = getQueryVariable("state");
-  if (!state1) {
-    window.location.replace(location.protocol + '//' + location.host + location.pathname +"?state=al");
-  }
-  $('select option[value="' + state1 + '"]')
-    .prop('selected', true);
+$(document.body).on("click", "[data-cmte-id]", function(e) {
+  e.preventDefault();
+  var $this = $(this);
+  var id = $this.data("cmte-id");
+  //rework with node querystring
+  // queryString.push('committee', committee_id);
 
-  populateDropDown(state1);
-  var sourceMembers = $("#committee-member-template")
-    .html();
-  app.memberTemplate = Handlebars.compile(sourceMembers);
-  var sourceMemberDetail = $("#committee-member-detail-template")
-    .html();
-  app.memberDetail = Handlebars.compile(sourceMemberDetail);
+  $this.closest("ul").css('left', '-99999px').removeClass("open");
+  $(".panel").html("Click a committee member for detail. <i class='fa fa-long-arrow-right'></i>");
+  getCommitteeDetail(id);
+});
+  
+$("select").on("change", function() {
+  var $this = $(this);
+  var selectedState = $(this).val();
+  // queryString.push('state', selectedState);
+  populateDropDown(selectedState);
```

Comment:
Because `populateDropDown` now replaces the HTML of the various menus, instead of prepending, we can call it repeatedly to change states instead of doing a hard refresh of the page.